extensions/ShowierData9978/html.js

```diff
@@ -0,0 +1,185 @@
+/* eslint-disable require-await */
```

Comment:
This extension is missing [required metadata](https://github.com/TurboWarp/extensions/blob/master/CONTRIBUTING.md#website-stuff) for all extensions within the repo.

Reply:
I don't think it's required for extensions that aren't listed on the homepage (and this PR does not seem to modify extensions.json).

Reply:
atleast right now, id rather get feedback before it being on the front page